upload.smk

```diff
@@ -0,0 +1,96 @@
+path_to_fauna = '../fauna'
+config["s3_dst"] = "s3://nextstrain-data-private/files/workflows/avian-flu"
+config["segments"] = ["pb2", "pb1", "pa", "ha","np", "na", "mp", "ns"]
+
+rule all:
+    input:
+        sequences=expand("upload/s3/sequences_{segment}.done", segment=config["segments"]),
+        metadata="upload/s3/metadata.done",
+
+rule download_segment:
+    output:
+        sequences = "upload/data/{segment}.fasta",
+    params:
+        fasta_fields = "strain virus accession collection_date region country division location host domestic_status subtype originating_lab submitting_lab authors PMID gisaid_clade h5_clade",
+    benchmark:
+        "benchmarks/download_segment_{segment}.txt"
+    shell:
+        """
+        python3 {path_to_fauna}/vdb/download.py \
+            --database vdb \
+            --virus avian_flu \
+            --fasta_fields {params.fasta_fields} \
+            --select  locus:{wildcards.segment} \
+            --path upload/data \
+            --fstem {wildcards.segment}
+        """
+
+rule parse_segment:
+    input:
+        sequences = "upload/data/{segment}.fasta",
+    output:
+        sequences = "upload/results/sequences_{segment}.fasta",
+        metadata = "upload/results/metadata_{segment}.tsv",
+    params:
+        fasta_fields =  "strain virus isolate_id date region country division location host domestic_status subtype originating_lab submitting_lab authors PMID gisaid_clade h5_clade",
```

Comment:
I dropped `PMID` in efb9e8b68e768e73307b1b74d4a541da03f3119b because it was almost fully empty. I think now would be the most appropriate time to slim columns from the S3 metadata that we're not actually using. Okay to drop here @lmoncla?

Reply:
The metadata entry for `h5_clade` is also very incomplete. It's not used as a coloring and instead we're using either GISAID clade or LABEL clade. This would seem to just add confusion. How about dropping this as well?

Reply:
Happy to leave this change to someone who knows the data better. I think we could merge this PR first and refine metadata in future commits/PRs, though.